plugins/query-insights/src/main/java/org/opensearch/plugin/insights/settings/QueryInsightsSettings.java

```diff
@@ -109,6 +111,37 @@ public class QueryInsightsSettings {
         Setting.Property.Dynamic
     );
 
+    /**
+     * Config key for exporter type
+     */
+    public static final String EXPORTER_TYPE = "type";
+    /**
+     * Config key for export index
+     */
+    public static final String EXPORT_INDEX = "config.index";
+
+    /**
+     * Settings and defaults for top queries exporters
+     */
+    public static final String TOP_N_LATENCY_QUERIES_EXPORTER_PREFIX = TOP_N_LATENCY_QUERIES_PREFIX + ".exporter.";
```

Comment:
This can be private. And, not sure if we really need it. Use it directly below might be slightly clearer